crates/key-server/src/types.rs

```diff
@@ -12,42 +15,69 @@ pub type MasterKeyPOP = ibe::ProofOfPossession;
 
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub enum Network {
-    Devnet,
+    Devnet {
+        seal_package: SealPackage,
```

Comment:
For Devnet the parameter is mandatory - for Custom we default to Mainnet as for MVR.